auctions/tests/model_tests.py

```diff
@@ -273,3 +289,14 @@ def test_save_updates_datetimes(self):
                         "Vote.modified should be auto-populated on update.")
         self.assertTrue(datetime.now() >= test_vote_modified,
                         "Vote.modified should be auto-populated.")
+
+    def test_user_claim_unique(self):
+        mommy.make(Vote, claim=self.claim, user=self.user1, approved=True)
+        with self.assertRaises(IntegrityError):
+            mommy.make(Vote, claim=self.claim, user=self.user1, approved=False)
+
+    @fudge.patch('auctions.models.send_mail')
+    def test_notify_claim_approved(self, mock_send_mail):
+        mock_send_mail.expects_call()
```

Comment:
Should fill in some more scenario tests here, or at least add a `# TODO: Add more test scenarios to VoteTest`